storage/.flake8

```diff
@@ -3,6 +3,7 @@ exclude =
   # Exclude generated code.
   **/proto/**
   **/gapic/**
+  **/types.py
```

Comment:
Not relevant for Storage and might actually clash with a manually-created file later.